airflow/providers/cncf/kubernetes/operators/spark_kubernetes.py

```diff
@@ -71,7 +71,7 @@ def execute(self, context):
             group=self.api_group,
             version=self.api_version,
             plural="sparkapplications",
-            body=self.application_file,
+            body=open(self.application_file).read(),
```

Comment:
Don’t do this. Use either `with open(...)` or `pathlib.Path(...).read_text()` instead.

Also, please specify `encoding="utf8")`.